src/implementations/twig/components/banner/banner.story.js

```diff
@@ -223,7 +224,10 @@ const prepareData = (data, args) => {
 
 const renderStory = (data, args) => {
   let story = banner(prepareData(data, args));
-  if (args.width === 'container' || args.width === 'inside') {
+  if (
+    args.width === 'inside the grid container' ||
```

Comment:
it felt there was something strange since you are receiving the labels and not the values in the args object, this is due to a reverse mapping, it should be:
```
    mapping: {
      'outside the ecl-container': 'outside',
      'inside the ecl-container': 'container',
      'inside the ecl-container, with fullwidth class': 'inside',
    },
```
and then you would deal with the values here, the same in the url ;)